core/client/src/main/java/alluxio/client/file/options/LoadMetadataOptions.java

```diff
@@ -19,9 +19,12 @@
 
 /**
  * Method options for loading the metadata.
+ *
+ * @deprecated since version 1.1 and will be removed in version 2.0
  */
 @PublicApi
 @NotThreadSafe
+@Deprecated
```

Comment:
we also want to mark loadMetadata in alluxio.client.FileSystem and alluxio.client.BaseFileSystem.java as deprecated